web-client/public/js/graph.js

```diff
@@ -1087,11 +1088,15 @@ export var drawGraph = function (workbook) {
                 .selectAll(".coloring")
                 .data(function () {
                     if (grnState.workbook.expression[dataset]) {
+                        var geneName = p.name;
+                        if (grnState.workbook.meta.data.workbookType === NETWORK_PPI_MODE && geneName.endsWith("p")) {
+                            geneName = geneName.slice(0, -1);
+                        }
```

Comment:
This sequence appears to be the same logic as the version in _update-app.js_ line 396 onward; as such, let’s streamline these to a more consistent approach between versions:

```suggestion
                        const geneName = (
                          grnState.workbook.meta.data.workbookType === NETWORK_PPI_MODE && geneName.endsWith("p")
                        ) ? p.name.slice(0, -1)
                          : p.name;
```

Reply:
Fixed